README.md

```diff
@@ -94,9 +94,7 @@ The supported algorithms can be extended at runtime by using [cose.RegisterAlgor
 
 CBOR supports integers in the range:
 
-```text
 [-2<sup>64</sup>, -1] ∪ [0, 2<sup>64</sup> - 1]
-```
```

Comment:
> CBOR supports integers in the range: [-2<sup>64</sup>, -1] ∪ [0, 2<sup>64</sup> - 1].

How about merging line 95 - 97? Would it be better?

Reply:
Sure!